src/main/java/core/basesyntax/service/FruitOperationService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface FruitOperationService {
+    int performOperation(int prev, int current);
+}
```

Comment:
```suggestion
public interface FruitOperationHandler {
    int handle(FruitOperation operation);
}
```

this handler should interact with dao and save new fruit quantity